src/server/index.test.jsx

```diff
@@ -17,6 +21,7 @@ import { NON_200_RESPONSE } from './utilities/customMetrics/metrics.const';
 
 // mimic the logic in `src/index.js` which imports the `server/index.jsx`
 dotenv.config({ path: './envConfig/local.env' });
+process.env.SENSITIVE_HTTP_HEADERS = 'x-sensitive-header,x-another-one';
```

Comment:
Could we add a test to cover when this parameter is missing? i.e. locally or if the cosmos config is missing. Just checking rendering still works and logging still works